src/modules/Common/components/Card.tsx

```diff
@@ -21,9 +25,13 @@ const Card: React.FC<CardProps> = ({
   image,
   subtitle,
   author,
+  authorIcon = true,
```

Comment:
not blocking but `authorIcon` let me think I can pass an icon of my own. Would have used `withAuthorIcon`

Reply:
+1, or `hasAuthorIcon`.